2.0/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gunicorn==19.7.1
-django==2.0.5
+django==2.0.6
 psycopg2==2.7.4 --no-binary psycopg2
 gevent==1.2.2
```

Comment:
Similar with Gevent (should ensure compatibility with Gunicorn, since it is a dependency for its async workers).

See: http://www.gevent.org/changelog.html